toolkit/partner-chains-cli/src/deregister/tests.rs

```diff
@@ -86,20 +103,16 @@ fn fails_when_cold_key_is_not_valid() {
 		.with_json_file(RESOURCES_CONFIG_FILE_PATH, test_resources_config_content())
 		.with_json_file(MY_PAYMEMENT_SKEY, valid_payment_signing_key_content())
 		.with_file(MY_COLD_VKEY, "not a proper Cardano key json")
-		.with_expected_io(vec![read_config_twice_io(), print_info_io(), read_keys_io()]);
+		.with_expected_io(vec![read_config_once_io(), print_info_io(), read_keys_io()]);
 	let result = DeregisterCmd.run(&mock_context);
 	assert_eq!(
 		result.err().unwrap().to_string(),
-		"my_cold.vkey is not a valid Cardano key file: expected ident at line 1 column 2"
+		r#"Failed to parse Cardano key file my_cold.vkey: Error("expected ident", line: 1, column: 2)"#
 	);
 }
 
-/// Read the chain configuration file twice, because for some reason cardano network is not present in ChainConfig.
-fn read_config_twice_io() -> MockIO {
-	MockIO::Group(vec![
-		MockIO::file_read(CHAIN_CONFIG_FILE_PATH),
-		MockIO::file_read(CHAIN_CONFIG_FILE_PATH),
-	])
+fn read_config_once_io() -> MockIO {
+	MockIO::Group(vec![MockIO::file_read(CHAIN_CONFIG_FILE_PATH)])
```

Comment:
This is very convoluted way of saying this thing.

Either leave just `MockIO::file_read(CHAIN_CONFIG_FILE_PATH)` or inline it call sites, please.